sdk/src/main/java/com/mapbox/maps/renderer/RenderHandlerThread.kt

```diff
@@ -39,8 +46,12 @@ internal class RenderHandlerThread {
     }
   }
 
-  fun clearMessageQueue() {
-    handler?.removeCallbacksAndMessages(null)
+  fun clearMessageQueue(clearAll: Boolean = true) {
```

Comment:
IMHO separate `clearSdkMessages` / `clearAllMessages` will be more clear, up to you